src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -45,13 +45,6 @@ public MutationTestInput Initialize(StrykerOptions options)
             // initial build
             _initialBuildProcess.InitialBuild(projectInfo.FullFramework, options.BasePath, options.SolutionPath, Path.GetFileName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath));
 
-            // If project is full framework, we need to reload projectinfo and rebuild
-            if (projectInfo.FullFramework)
```

Comment:
Have you tested stryker with this removed? We have not been able to get buildalyzer to recognize all project dependencies without building and analyzing twice, because buildalyzer cleans during build.

Reply:
I was wondering why this was in there :) I've run multiple sessions after removing this and did not notice any difference. I will restore this though.

Reply:
Restored